hypothesis-python/docs/extras.rst

```diff
@@ -1,9 +1,9 @@
-===================
-Additional packages
-===================
+======================
+First-party extensions
+======================
 
-Hypothesis itself does not have any dependencies, but there are some packages that
-need additional things installed in order to work.
+Hypothesis itself only depends on :pypi:`attrs`, so we provide separate ``extra``
```

Comment:
Maybe worth replacing with something like "Hypothesis tries to keep its dependency footprint small" so we don't have to change this every time we add a dependency?

Reply:
Given the low frequency of change, I'd rather be specific and take the two minutes to edit this again if we add another dependency - which we won't do lightly anyway.